test/commands/deleteEnvironmentCommand.test.ts

```diff
@@ -105,7 +105,42 @@ describe('DeleteEnvironmentCommand', () => {
             logSpy.getCall(1).should.have.been.calledWithExactly(LogType.SUCCESS, `Successfully deleted ${myEnvironmentB.name} environment`);
         });
 
-        it('should test a environment can be deleted from tree', async () => {
+        it('should test multiple environments can be deleted from the command', async () => {
+            showFabricEnvironmentQuickPickBoxStub.resolves([{
+                label: 'myenvironmentA',
+                data: FabricEnvironmentRegistry.instance().get('myenvironmentA')
+            }, {
+                label: 'myEnvironmentB',
+                data: FabricEnvironmentRegistry.instance().get('myEnvironmentB')
+            }]);
+
+            await vscode.commands.executeCommand(ExtensionCommands.DELETE_ENVIRONMENT);
+
+            environments = vscode.workspace.getConfiguration().get(SettingConfigurations.FABRIC_ENVIRONMENTS);
+            environments.length.should.equal(0);
+
+            logSpy.getCall(0).should.have.been.calledWithExactly(LogType.INFO, undefined, `delete environment`);
+            logSpy.getCall(1).should.have.been.calledWithExactly(LogType.SUCCESS, `Successfully deleted environments`);
+        });
+
+        it('should test that the only non-local environment can be deleted from the command', async () => {
+            await vscode.workspace.getConfiguration().update(SettingConfigurations.FABRIC_ENVIRONMENTS, [{name: 'myenvironmentA'}], vscode.ConfigurationTarget.Global);
```

Comment:
There is already a test to make sure you can't delete the local environment, all this test does is test that you can delete a non local one which we have already????